internal/pkg/deploy/cloudformation/ghV1_pipeline_integration_test.go

```diff
@@ -142,6 +146,22 @@ func TestGHv1PipelineCreation(t *testing.T) {
 		})
 		require.NoError(t, err)
 
+		err = appDeployer.AddEnvToApp(&cloudformation.AddEnvToAppOpts{
+			App:          &app,
+			EnvName:      environmentToDeploy.Name,
+			EnvRegion:    envRegion.ID(),
+			EnvAccountID: envCallerInfo.Account,
+		})
+		require.NoError(t, err)
+
+		regionalResource, err := appDeployer.GetAppResourcesByRegion(&app, envRegion.ID())
+		require.NoError(t, err)
+		urls, err := uploader.UploadEnvironmentCustomResources(s3.CompressAndUploadFunc(func(key string, objects ...s3.NamedBinary) (string, error) {
+			return s3Client.ZipAndUpload(regionalResource.S3Bucket, key, objects...)
+		}))
+		require.NoError(t, err)
+		environmentToDeploy.CustomResourcesURLs = urls
```

Comment:
We really need to remove some duplication from our integ tests and add some parallelism:
```go
t.Run("create app", func(t *testing.T) {
   t.Run("create env", func(t * testing.T) {
        t.Run("gh pipeline", func(t *testing.T) {
             t.Parallel()
        })
        t.Run("gh v2 pipeline", func(t *testing.T) {
              t.Parallel()
        })
        t.Run("lb web service", func(t *testing.T) {
              t.Parallel()
        })
   })
})
```

Writing it here to so that we can keep it in mind next time we work on integ tests (no need to change it now) :D 
I think this change would be amazing  and make it overall simpler for our team to add new tests